bindata/manifests/webhook/server.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: network-resources-injector
+  name: network-resources-injector
+  namespace: {{.Namespace}}
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the network resource injector component on master nodes.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  selector:
+    matchLabels:
+      app: network-resources-injector
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: network-resources-injector
+        component: network
+        type: infra
+        openshift.io/component: network
+    spec:
+      serviceAccountName: network-resources-injector-sa
+      tolerations:
+      - key: "node-role.kubernetes.io/master"
+        operator: Exists
+        effect: NoSchedule
+      - key: "node.kubernetes.io/not-ready"
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: webhook-server
+        image: {{.NetworkResourcesInjectorImage}}
+        command:
+        - webhook
+        args:
+        - -bind-address=0.0.0.0
+        - -port=443
+        - -tls-private-key-file=/etc/tls/tls.key
+        - -tls-cert-file=/etc/tls/tls.crt
```

Comment:
Good point!
This will require further testing and might be solved in two ways:
1) admission controller reload the `tls.key` and `tls.crt` upon any change.
2) operator monitor change of secret and restart admission controller.